go/oasis-node/cmd/common/signer/signer.go

```diff
@@ -33,6 +35,8 @@ const (
 	cfgSignerRemoteClientCert = "signer.remote.client.certificate"
 	cfgSignerRemoteClientKey  = "signer.remote.client.key"
 	cfgSignerRemoteServerCert = "signer.remote.server.certificate"
+
+	cfgSignerCompositeBackends = "signer.composite.backends"
```

Comment:
Can we have a test for this argument in E2E tests?

Reply:
That's mostly useless at the moment.  I added unit tests for the ctor instead, please take a look.

Reply:
:heavy_check_mark: